src/dmd/backend/backend.txt

```diff
@@ -1,19 +1,6 @@
 Files in the Back End
 =====================
 
-aa.c            simple hash table
-aa.h            header for simple hash table
-bcomplex.c      our own complex number implementation because we can't rely on host C compiler
-bcomplex.h      header for our own complex numbers
-blockopt.c      manage and simple optimizations on graphs of basic blocks
-cc.h            main header file for back end
-cdef.h          configuration
-cg.c            global variables for code generator
-cg87.c          x87 FPU code generation
-cgcod.c         main loop for code generator
-cgcs.c          compute common subexpressions for non-optimized code generation
-cgcv.c          CodeView symbol debug info generation
-cgcv.h          header for cgcv.c
```

Comment:
Is there a particular reason for removing this? 
Its pretty useful for the occasional backend stuff, because I don't go there often enough to remember where things are and the module names are ... pretty cryptic.

They should be `.d` files though.

Reply:
We don't need a documentation header at the top of each .d file *and* backend.txt.  Using a documentation header is the typical, standard way of documenting D code, so I'm proposing that.  

Furthermore, if we do both, they're just going to get out of sync, much like that are right now.  And the fact that backend.txt has been in this state for so long demonstrates that no one's actually looking at it.

Reply:
I don't see a problem with having, in effect, a Table of Contents. I could flip through a book to find the chapter names, but a ToC is most convenient. I suggest just updating it to reflect the D conversion.

Reply:
And actually, I think we should be doing more of this for the other directories.

Reply:
Even better, make the `cg87.d` a hyperlink to the file in github, and change it to a Markdown file.

Reply:
I think if users want a table of contents, they should look to the published HTML documentation.

![image](https://user-images.githubusercontent.com/6023641/55677016-6aa99b00-591b-11e9-8a65-a933acdaf33c.png)

I ask you to please reconsider deleting backend.txt in favor of our official published documentation generated from DDoc.